### PR TITLE
Add notes on practicalities when the sole technologist on a project

### PR DESCRIPTION
**Pull request by lozette**
Add a section about practicalities with the commit/review/merge cycle when a developer is the sole technologist on a project.

### CODE REVIEW

File: src/tech/how-we-do-development.md
Comment (yndajas) [inline src/tech/how-we-do-development.md:418]: "commit/review/merge cycle" -> this seems more focused on reviewing specifically. Are there impacts on commits and merges as well?

File: src/tech/how-we-do-development.md
Comment (yndajas) [inline src/tech/how-we-do-development.md:432]: dxw developer and solo developer generally not being mutually exclusive descriptions confused me a little here. Small wording suggestion below to clarify that and something else:

```suggestion
If the client does not have anyone suitable, a dxw developer can review pull requests for the solo developer. If the repository belongs to the client, the reviewer should be granted access to merge pull requests, *or* the solo developer should be allowed to merge their own pull requests. The client should be made aware of this requirement, should their security policies not allow developers to merge their own pull requests.
```

Also:
- "the dxw developer should be granted access to merge pull requests" - is it more that they should be granted write access if the repository requires PR approval from someone with write access? I would've thought the person opening the PR would typically be responsible for merging regardless of the review process?
- "the solo developer should be allowed to merge their own pull requests" - without requiring approval? I'd guess merging their own pull requests in general is not a problem, it's just the approval part?

**Review by yndajas — APPROVED** [review]
Generally looks good to me, but a few suggestions

File: src/tech/how-we-do-development.md
Comment (lozette) [reply src/tech/how-we-do-development.md:418]: There are impacts on merges when a client's security policy disallows developers merging their own commits (DfE does this for example). I'll take out "commit" though, you're right there.

Comment (lozette) [reply src/tech/how-we-do-development.md:418]: You're right I'm getting confused with approvals with write access vs approvals with no write access 🤦

**Pull request merged**
